support for keyfinder-cli, which doesn't want the -f flag for paths (#3467)
beetbox/beets: docs/plugins/keyfinder.rst

Key Finder Plugin
=================

> The `keyfinder` plugin uses either the `KeyFinder`_ or `keyfindercli`_ program

Reviewer: ```suggestion
The `keyfinder` plugin uses either `KeyFinder`_ or `keyfinder-cli`_
```

> to  detect the musical key of track from its audio data and store it in the

Reviewer: ```suggestion
to  detect the musical key of a track from its audio data and store it in the
```

>   a path to the binary. If you installed the KeyFinder GUI on a Mac, for

Reviewer: Perhaps the sentence above should read:

> The name of the program use for key analysis. You can use either 
\`KeyFinder\`_ or \`keyfinder-cli\`_.

  example, you want something like
  ``/Applications/KeyFinder.app/Contents/MacOS/KeyFinder``.

>   If using `keyfindercli`_, the binary must be named keyfinder-cli.

Reviewer: ```suggestion
  If using `keyfinder-cli`_, the binary must be named ``keyfinder-cli``.
```
